src/measure/OSRunner.hpp

```diff
@@ -274,6 +274,11 @@ namespace measure {
 
     void resetLanguagePreference();
 
+    /** Whether registerInfo, registerWarning, registerError, registerInitialCondition, registerFinalCondition and registerAsNotApplicable
+      * also LOG an actual message */
+    bool registerMsgAlsoLogs() const;
+    void setRegisterMsgAlsoLogs(bool registerMsgAlsoLogs);
```

Comment:
Here

Reply:
I feel like option like this is just punting on making a decision. I think I would prefer that we just make a call one way or the other and avoid giving an option like this.

Reply:
Until we remove the Ruby CLI (classic), we need an option, otherwise if always enabled, it'll log TWICE int he ruby cli

Reply:
ok.